content/03-reference/01-tools-and-interfaces/02-prisma-client/08-pagination/index.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: 'Pagination'
+metaTitle: 'Pagination (Reference)'
+metaDescription: 'Prisma Client provides features that are not achievable with relational databases. These features are referred to as "polyfills" and explained on this page.'
+---
+
+## Overview
+
+Prisma Client supports both offset and cursor-based pagination.
```

Comment:
```suggestion
Prisma Client supports both offset- and cursor-based pagination.
```
Not sure if this is correct.

Reply:
Changed to 'offset pagination and cursor-based pagination'.